tests/page/to-match-aria-snapshot.spec.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { test, expect, role as x } from './pageTest';
+
```

Comment:
Let's add more tests, or at least TODOs:
- `<x.link />` should match `<a href='.'>some text</a>`
- shadow dom test
- `role=presentation`
- skipping medium levels in the template
- non-body locator

Reply:
I believe we will need many more